Ginger/GingerCoreNET/Database/ActDBValidation.cs

```diff
@@ -311,7 +311,22 @@ private void HandleNoSQLDBAction()
                         }
                     }
                     break;
+                case Database.eDBTypes.CosmosDb:
+
+                    if (NoSqlDriver == null)
+                    {
+                        NoSqlDriver = new GingerCosmos(DBValidationType, DB, this);
+                    }
+                    else
+                    {
+                        if (NoSqlDriver.GetType() != typeof(GingerMongoDb))
```

Comment:
add or condition instead of 2 If statements

Reply:
done